### ISSUE TITLE
Disabled categories show up in search results

### ISSUE THREAD

**Issue by peterhirn** [Smile-SA/elasticsuite#292]
Setting `Use Category Name in product search` has no effect.

**Comment by afoucret** [comment]
This settings has no effect in the autocomplete of categories but indicates if the name of categories should be added to the product search data.

But you are right, disabled categories are shown in the autocomplete.

The PR #295 will fix this one.

**Comment by afoucret** [comment]
You can test the patch available in the branches 2.3.x and master.
Warning : this patch require the `elasticsuite_categories_fulltext` indexer to be run :

```
bin/magento indexer:reindex elasticsuite_categories_fulltext
```

It will be part of the next minor release (2.3.1)
